src/context/AppProvider/AppContext.tsx

```diff
@@ -154,16 +153,16 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
 
   // once portfolio is done loading, fetch all transaction history
   useEffect(() => {
-    if (!isPortfolioLoaded) return
```

Comment:
Not needed in this PR but we'll probably want to update all occurences of `selectPortfolioLoading` as well